Make improvements to building a data service guide (#5037)
ballerina-platform/ballerina-dev-website: learn/get-started/build-a-data-service-in-ballerina.md

   artifactId = "mysql-connector-java"
   version = "8.0.26"
   ```
   >**Info:** This guide was tested using MySQL driver version 8.0.26. However, any MySQL driver versions after 8.0.26
   >should also work the same.
   
3. [Download](https://dev.mysql.com/downloads/connector/j/) the driver JAR manually and update the path in the 
   `Ballerina.toml` file
   ```toml
   # In Unix operating systems
   [[platform.java11.dependency]]
   path = "path/to/mysql/driver.jar”
   
   # In Windows operating systems
   [[platform.java11.dependency]]
   path = "path\to\mysql\driver.jar”
   ```
   
### Define MySQL configurations

> In the newly created directory `"data_service"` (using the `bal new` command), create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.

Reviewer: ```suggestion
In the  `data_service` directory (you created above using the `bal new` command), create a new file named `Config.toml`, and specify the configurations below to connect to the MySQL database.
```

Author: Fixed [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

To use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database, add the code below to the `main.bal` file. 

```ballerina

> import ballerina/sql;

Reviewer: ```suggestion
import ballerina/sql;
```
Don't we need to remove this unused import?

Author: The unused import was in a previous sample. It has been removed from there.

https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/files#diff-439f92e0c6479c601f714f2743e690239e2745cf2eaecd4de77817f7c9ad9836L169

Author: Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

You view the output below.

>**Info:** This creates an `/employees` endpoint on port `8080`, which can 

> be accessed via a browser by visiting `http://localhost:8080/employees`.

Reviewer: ```suggestion
be accessed by visiting `http://localhost:8080/employees`.
```

Author: Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

## Try the service

Invoke the defined resource function by sending the `POST` request below to `http://localhost:8080/employees` with the required data as a JSON payload.

> >**Info:** The following CURL command only works with Unix operating systems.

Reviewer: ```suggestion
>**Info:** The following cURL command only works with Unix operating systems.
```
Can we give the command for Windows as well in the info section saying, "If you are using the Windows OS, execute this cURL command: <command>."
